fem/gridfunc.hpp

```diff
@@ -94,13 +99,28 @@ class GridFunction : public Vector
 
    int VectorDim() const;
 
+   /// Read only access to the (optional) internal true-dof Vector.
```

Comment:
I would prefer these methods be labeled "TrueVector" or something like that to avoid confusion with the template classes and generally be more descriptive.